__tests__/db.test.js

```diff
@@ -35,6 +35,62 @@ describe("GET /api/topics", () =>{
     })
 })
 
+describe("GET /api/articles/:article_id/comments", () =>{
+    test("Returns 200 status code with an array", () =>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then((res)=>{
+            expect(res.statusCode).toBe(200)
+            expect(Array.isArray(res.body))
+        })
+    })
+    test("Returns an array with appropriate data", ()=>{
+        return request(app)
+        .get("/api/articles/1/comments")
+        .then(({body})=>{
+            body.forEach((comment)=>{
```

Comment:
Ccheck the length of the array that is passed back  before the forEach. A foreach can produce a false positive if passed an empty array :)